src/js/constants.js

```diff
@@ -32,6 +34,7 @@ export const MediaStateChangeEvents = {
   MEDIA_VOLUME: 'mediavolumechange',
   MEDIA_VOLUME_UNAVAILABLE: 'mediavolumeunavailablechange',
   MEDIA_IS_PIP: 'mediaispipchange',
+  MEDIA_IS_CAST: 'mediaiscastchange',
```

Comment:
I see how we got here with IS_PIP, but IS_CAST is awkward phrasing. I think we need to consider _casting_ or _casted_ or something similar. I'm not sure what's best off the top of my head.

Reply:
renamed it to MEDIA_IS_CASTING